### PR TITLE
LPS-92008 portal-search-web: Set visibility in search widgets

### PR DESCRIPTION
**Pull request by oliv-yu**
https://issues.liferay.com/browse/LPS-81279
https://issues.liferay.com/browse/LPS-92008

Added to the sort portlet as well :) Let me know if this works as intended!

@jonmak08

### CODE REVIEW

**Comment by oliv-yu** [comment]
ci:test:sf:norebase

**Comment by oliv-yu** [comment]
ci:test:search:norebase

File: modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/custom/filter/portlet/CustomFilterPortlet.java [hunk 1]
```diff
@@ -86,7 +88,7 @@ public void render(
 		renderRequest.setAttribute(
 			WebKeys.PORTLET_DISPLAY_CONTEXT, customFilterDisplayContext);
 
-		if (customFilterPortletPreferences.isDisabled() ||
+		if (customFilterDisplayContext.isRenderNothing() ||
 			customFilterPortletPreferences.isImmutable() ||
 			customFilterPortletPreferences.isInvisible()) {
```
Comment (thektan) [inline modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/custom/filter/portlet/CustomFilterPortlet.java]: Let's re-alphabetize these checks.

File: modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/suggestions/portlet/SuggestionsPortlet.java [hunk 2]
```diff
@@ -91,6 +91,13 @@ public void render(
 		renderRequest.setAttribute(
 			WebKeys.PORTLET_DISPLAY_CONTEXT, suggestionsPortletDisplayContext);
 
+		if (!suggestionsPortletDisplayContext.hasSpellCheckSuggestion() &&
+			!suggestionsPortletDisplayContext.hasRelatedQueriesSuggestions()) {
+
+			renderRequest.setAttribute(
+				WebKeys.PORTLET_CONFIGURATOR_VISIBILITY, Boolean.TRUE);
+		}
```
Comment (thektan) [inline modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/suggestions/portlet/SuggestionsPortlet.java]: Let's alphabetize:
1) `if` check logic
2) `WebKeys.PORTLET_CONFIGURATOR_VISIBILITY` and `WebKeys.PORTLET_DISPLAY_CONTEXT`

File: modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/search/results/portlet/SearchResultsPortlet.java [hunk 3]
```diff
@@ -117,6 +117,11 @@ public void render(
 			WebKeys.PORTLET_DISPLAY_CONTEXT,
 			searchResultsPortletDisplayContext);
 
+		if (searchResultsPortletDisplayContext.isRenderNothing()) {
+			renderRequest.setAttribute(
+				WebKeys.PORTLET_CONFIGURATOR_VISIBILITY, Boolean.TRUE);
+		}
```
Comment (thektan) [inline modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/search/results/portlet/SearchResultsPortlet.java]: Alphabetize (same as above)

File: modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/modified/facet/portlet/ModifiedFacetPortlet.java [hunk 4]
```diff
@@ -94,6 +94,11 @@ public void render(
 		renderRequest.setAttribute(
 			WebKeys.PORTLET_DISPLAY_CONTEXT, modifiedFacetDisplayContext);
 
+		if (modifiedFacetDisplayContext.isRenderNothing()) {
+			renderRequest.setAttribute(
+				WebKeys.PORTLET_CONFIGURATOR_VISIBILITY, Boolean.TRUE);
+		}
```
Comment (thektan) [inline modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/modified/facet/portlet/ModifiedFacetPortlet.java]: Let's move this before the `renderRequest.setAttribute(WebKeys.PORTLET_DISPLAY_CONTEXT...` to alphabetize the attributes.

**Comment by oliv-yu** [comment]
@thektan Thank you!! Added the commit to alphabetize

**Comment by thektan** [comment]
ci:test:sf

**Comment by thektan** [comment]
ci:test:search:norebase

**Comment by thektan** [comment]
Sent with QA changes: https://github.com/brianchandotcom/liferay-portal/pull/86265

**Pull request closed without merging**